### PR TITLE
[UI Cleanup] - Small changes across site

### LINKED ISSUE #115: [UI Feature] - Small changes across site
### Description

As a developer, I want to be able to clean up small UI changes so that my users have a good experience on the site.

### Acceptance Criteria

- [ ] Group subscription card style updates
- [ ] Fix row/col stacking for mobile devices
- [ ] Tables scrolling on overflow
- [ ] Cucumber and Rspec fixes

### PR DESCRIPTION
**Pull request by coraenglish**
### Pivotal Tracker
- https://www.pivotaltracker.com/story/show/182006411

### Description / PR Commit Message

- Cleanups that were too small for individual PRs:
- Added fancier G logo to navbar and fixed onload function when clicked
- Fixed column stacking for subscriptions, groups, reminders, and statistics
- Added styling to notices

### Guidance for Review

- Cucumber tests fixed 
- Did not test deployment

### Author's Checklist
- [x] Fixed tests?
- [x] Linked to issue in Github?
- [x] Tests passed?
- [x] Test deployment to gatekeeper-tamu-test?

### CODE REVIEW

File: app/views/statistics/_group.html.erb [hunk 1]
```diff
@@ -41,13 +41,13 @@
   <% if current_user.all_groups.count > 0 %>
 
     <div class="row stat-row">
-      <div class="col col-lg-4 stat-col-info group-color">
+      <div class="col-md-4 stat-col-info group-color">
         <h4 id="stat-group-header">Montly Cost for Group <%= current_user.all_groups.first().group_name %></h4>
         <p>The total monthly cost for this group is <b id="stat-group-cost">$<%= current_user.group_cost(current_user.all_groups.first()) %></b>.
         Consider splitting this cost among all of your group members that use the accounts listed here in order to share the burden of paying for these services.
         Obviously, you don't have to do this, but your the other members of your group might like you more if you do 😉</p>
```
Comment (rebeccamcfadden) [inline app/views/statistics/_group.html.erb:48]: lol

File: db/seeds.rb [hunk 2]
```diff
@@ -47,7 +47,7 @@
 hermione = User.where(username: "hogwartsahistory").first
 hermione.subscriptions.create([
     {subscription_name: "Flourish and Blotts", username: "hermione", password: "wingardiumleviOsa", url: "flourish.com", cost_per_month: 4.99},
-    {subscription_name: "Hogwarts Tuition Portal", username: "granger", password: "password", url: "hogwarts.com", cost_per_month: 1495.95}
+    {subscription_name: "Hogwarts Tuition Portal", username: "granger", password: "password", url: "hogwarts.com", cost_per_month: 105.95}
```
Comment (rebeccamcfadden) [inline db/seeds.rb:50]: wow discounted hogwarts

**Review by rebeccamcfadden — APPROVED** [review]
LGTM! I'll test the deployment and ping you when it's up. Feel free to merge if your changes are reflected on heroku

File: db/seeds.rb (hunk 2 above)
Comment (coraenglish) [reply db/seeds.rb:50]: yes I did haha. can't have that huge bar dwarfing all of the other things

**Pull request merged**